lottie-swift/src/Public/iOS/Compatibility/CompatibleAnimationView.swift

```diff
@@ -245,6 +294,23 @@ public final class CompatibleAnimationView: UIView {
 
     return UIColor(red: CGFloat(colorValue.r), green: CGFloat(colorValue.g), blue: CGFloat(colorValue.b), alpha: CGFloat(colorValue.a))
   }
+  
+  
+  @objc
+  public func setFloatValue(_ value: CGFloat, forKeypath keypath: CompatibleAnimationKeypath)
+  {
+    
+    let valueProvider = FloatValueProvider(value)
+    animationView.setValueProvider(valueProvider, keypath: keypath.animationKeypath)
+  }
+
+  @objc
+  public func getFloatValue(for keypath: CompatibleAnimationKeypath, atFrame: CGFloat) -> NSNumber?
+  {
```

Comment:
I think the default style for opening braces is preceded by a single space and on the same line as the declaration:
```swift
 @objc
  public func setFloatValue(_ value: CGFloat, forKeypath keypath: CompatibleAnimationKeypath) {
    let valueProvider = FloatValueProvider(value)
    animationView.setValueProvider(valueProvider, keypath: keypath.animationKeypath)
  }

  @objc
  public func getFloatValue(for keypath: CompatibleAnimationKeypath, atFrame: CGFloat) -> NSNumber? {
    let value = animationView.getValue(for: keypath.animationKeypath, atFrame: atFrame)
    return value as? NSNumber
  }
```